store/storetest/user_store.go

```diff
@@ -3377,8 +3377,12 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 	// Ensure update at timestamp changes
 	time.Sleep(time.Millisecond * 10)
 
-	store.Must(ss.Team().RemoveMember(teamId, u1.Id))
-	store.Must(ss.Team().RemoveMember(teamId, u2.Id))
+	e := ss.Team().RemoveMember(teamId, u1.Id)
+	require.Nil(t, e)
+
+	e = ss.Team().RemoveMember(teamId, u1.Id)
```

Comment:
You have a typo here, `u1` should be `u2`, probably that is the reason why the tests are failing.

Reply:
@jespino thank you for the review and guidance. I have made the changes and I think the tests are now passing just fine. I am requesting for an extra review of my PR as this is my first contribution. :)

Reply:
Sure, no problem 😊. Welcome to the project! :dancer: